app/_data/docs_nav_konnect.yml

```diff
@@ -204,19 +204,22 @@
   items:
     - text: Introduction to Analytics
       url: /analytics/
-    - text: Report Use Cases
+    - text: Dashboard
```

Comment:
I thought it might be good to prep for when we actually have dashboards with customization features + creating your own. 

That allows us to avoid renaming and redirecting in the future if we call it "Dashboards" now and not "Dashboard" (singular). I am fine that only the summary dashboard is in that section for now. Is that a concern for you?